.github/workflows/codeql-config.yml

```diff
@@ -0,0 +1,6 @@
+paths:
+  - packages
+paths-ignore:
+  - packages/**/node_modules
+  - packages/**/dist
+  - '**/*.spec.ts'
```

Comment:
When discussing with the GHAS colleagues it seemed very obvious for me, that static security checks should be running on tests as well. Currently I can't make up good examples why that was. Anyways, how much longer does it take with test files?